src/ophys_etl/decrosstalk/decrosstalk_schema.py

```diff
@@ -0,0 +1,121 @@
+import marshmallow.exceptions
+import argschema
+
+
+class RoiSchema(argschema.ArgSchema):
+    id = argschema.fields.Int(description='ID', required=True)
+    x = argschema.fields.Int(description='x origin', required=True)
+    y = argschema.fields.Int(description='y origin', required=True)
+    width = argschema.fields.Int(description='width', required=True)
+    height = argschema.fields.Int(description='height', required=True)
+    valid_roi = argschema.fields.Bool(description='validity', required=True)
+    mask_matrix = argschema.fields.List(
+                        argschema.fields.List(argschema.fields.Bool),
+                        description='mask',
+                        cli_as_single_argument=True)
+
+
+class PlaneSchema(argschema.ArgSchema):
+
+    ophys_experiment_id = argschema.fields.Int(
+                             description='experiment ID',
+                             required=True)
+
+    motion_corrected_stack = argschema.fields.Str(
+                        description='path to motion corrected movie',
+                        required=True)
+
+    motion_border = argschema.fields.Dict(description='motion border',
+                                          required=True)
+
+    rois = argschema.fields.List(
+                argschema.fields.Nested(RoiSchema),
+                description='regions of interest',
+                required=True,
+                cli_as_single_argument=True)
+
+
+class PlanePairSchema(argschema.ArgSchema):
+
+    ophys_imaging_plane_group_id = argschema.fields.Int(
+                                    description='group ID',
+                                    required=True)
+
+    group_order = argschema.fields.Int(
+                      description='order of group in session',
+                      required=True)
+
+    planes = argschema.fields.List(
```

Comment:
you might also consider Nested(many=True)
https://marshmallow.readthedocs.io/en/stable/marshmallow.fields.html#marshmallow.fields.Nested
example:
https://github.com/AllenInstitute/ophys_etl_pipelines/blob/1225bd042c8e51598dbbfd30ca26ab6b27f276e1/src/ophys_etl/transforms/classification.py#L233-L237